CollectionView/UI/Cell/MyCollectionViewCell.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  MyCollectionViewCell.swift
+//  CollectionView
+//
+//  Created by Felipe Gil on 2021-08-23.
+//
+
+import UIKit
+
+class MyCollectionViewCell: UICollectionViewCell {
+    @IBOutlet var imageView: UIImageView!
+    static let identifier = "MyCollectionViewCell"
+    var targetURL = ""
+    var task: DispatchWorkItem?
+ 
+    override func prepareForReuse() {
+        imageView.image = nil
+        task?.cancel()
+    }
+    
+    public func configure(with stringURL: String?, with stringId: Int?) {
+        
+        var imageData: Data?
```

Comment:
`imageData` is being used only inside the task, so I'd move its declaration there.